stellargraph/layer/gcn.py

```diff
@@ -27,8 +27,28 @@
 
 
 class GraphConvolution(Layer):
+
     """
-    Implementation of the graph convolution layer as in https://arxiv.org/abs/1609.02907
+    Graph Convolution (GCN) Keras layer.
+    The implementation is based on the keras-gcn github repo https://github.com/tkipf/keras-gcn.
+
+    Original paper: Semi-Supervised Classification with Graph Convolutional Networks. Thomas N. Kipf, Max Welling,
+    International Conference on Learning Representations (ICLR), 2017 https://github.com/tkipf/gcn
+
+    Args:
+        units (int): dimensionality of output feature vectors
+        support (int): number of support weights
+        activation (str): nonlinear activation applied to layer's output to obtain output features
+        use_bias (bool): toggles an optional bias
+        kernel_initializer (str): name of layer bias f the initializer for kernel parameters (weights)
+        bias_initializer (str): name of the initializer for bias
+        attn_kernel_initializer (str): name of the initializer for attention kernel
+        kernel_regularizer (str): name of regularizer to be applied to layer kernel. Must be a Keras regularizer.
+        bias_regularizer (str): name of regularizer to be applied to layer bias. Must be a Keras regularizer.
+        activity_regularizer (str): not used in the current implementation
+        kernel_constraint (str): constraint applied to layer's kernel
+        bias_constraint (str): constraint applied to layer's bias
+        **kwargs:
```

Comment:
description of **kwargs?

Reply:
Based on the practices of keras layer code https://github.com/keras-team/keras/blob/master/keras/layers/core.py, we should remove it from comments.